.readthedocs.yml

```diff
@@ -1,4 +1,20 @@
-requirements_file: requirements.txt
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+version: 2
 
+sphinx:
+  builder: html
+  configuration: docs/conf.py
+  fail_on_warning: true
+
+# Set the version of Python and requirements required to build your docs
 python:
-   version: 3.5
+  version: 3.7
+  system_packages: true
+  install:
+    - requirements: requirements.txt
+    - method: setuptools
```

Comment:
why `setuptools` and not `pip`?

Reply:
To run `python setup.py install`. I think @astrofrog already brought it up above and said the `pip` thing can be a separate PR.